app/controllers/users_controller.rb

```diff
@@ -70,6 +79,32 @@ def members_index
     render json: UserSerializer.new(@members, @serializer_options).serialized_json
   end
 
+  # GET /users?serch=querystring&not_in_project=:id
+  def search_users
+    querystring = params[:search]
+
+    if querystring.present?
+      if params[:not_in_project].present?
```

Comment:
questo parametro lo chiamerei semplicemente ```:project_id``` perchè è effettivamente ciò che stiamo cercando

Reply:
Però a me con ```:not_in_project``` viene in mente un utente, non il parametro di ricerca per un progetto, so...